docs/tutorial/index.md

```diff
@@ -1,36 +1,29 @@
 (tutorial)=
-# Getting-Started Tutorial
+# Tutorials and Examples
 
-This tutorial provides a hands-on introduction to using MLRun to implement a data science workflow and automate machine-learning operations (MLOps).
+The following tutorials provide a hands-on introduction to using MLRun to implement a data science workflow and automate machine-learning operations (MLOps).
 
-The tutorial covers MLRun fundamentals such as creation of projects and data ingestion and preparation, and demonstrates how to create an end-to-end machine-learning (ML) pipeline.
-MLRun is integrated as a default (pre-deployed) shared service in the Iguazio MLOps Platform.
+Make sure you start with the [**Quick Start Tutorial**](./01-mlrun-basics.ipynb) to understand the basics.
 
-You can see another example which demonstrate how to [**Convert Research Notebook (NYC Taxi) to Operational Pipeline**](../howto/convert-to-mlrun.md).
-For additional demos refer to MLRun demos repository at [github.com/mlrun/demos](https://github.com/mlrun/demos).
+Each of the following tutorials is a dedicated Jupyter notebook, it can be downloaded by clicking on the `download` icon at the top.
```

Comment:
Each of the following tutorials is a dedicated Jupyter notebook. You can download them by clicking the `download` icon at the top of each page.